x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/steps/data_stream_step/sample_logs_input.tsx

```diff
@@ -143,44 +24,35 @@ export const SampleLogsInput = React.memo<SampleLogsInputProps>(({ integrationSe
       const logsSampleFile = files?.[0];
       if (logsSampleFile == null) {
         setSampleFileError(undefined);
-        setIntegrationSettings({
-          ...integrationSettings,
-          logsSampleParsed: undefined,
-          samplesFormat: undefined,
-        });
+        setIntegrationSettings({ ...integrationSettings, encodedLogSamples: undefined });
+        return;
+      }
+
+      if (logsSampleFile.size > 9000000) {
```

Comment:
Out of curiosity, where does the limit come from? Is it an arbitrary limit that we have defined to avoid processing to much data, or is it due to some limitation?

Reply:
Good question... Some background in this PR - https://github.com/elastic/kibana/pull/72898

`Kibana with 1 gig of memory can upload ~10 megs of a list before possible out of memory issue
`

Reply:
Can the value be set to a named constant to avoid a magic number and add some of this context?